src/forms/SubmissionListSidebar.js

```diff
@@ -32,6 +33,9 @@ export default class Sidebar extends Component {
 
       const {activeSubmission, submissions, onSelect, onFlag, onBookmark} = this.props;
 
+      // if the search input has focus, don't do anything.
+      if (document.activeElement === ReactDOM.findDOMNode(this.refs.searchInput)) return;
```

Comment:
Is there any way to do this more universally?  I foresee this being an issue whenever we use keyboard shortcuts on any page with text input.